client/admin/notificationUtils/notificationAdd/notificationAdd.jsx

```diff
@@ -0,0 +1,158 @@
+require('./notificationAdd.less');
+const React = require('react');
+const { useState, useRef } = require('react');
+const cx = require('classnames');
```

Comment:
cx should not be needed; this was a limitation of old React but you can just concatenate classes in a string now.